docs/main/user-guides/upgrades.md

```diff
@@ -29,189 +29,72 @@ This upgrade process allows you to upgrade to the latest OpenEBS version 4.2 whi
 and one Replicated Storage (a.k.a Replicated Engine):
 - Replicated PV Mayastor
 
-As a part of the upgrade to OpenEBS 4.2, the helm chart will install all four engines irrespective of the engine you used before the upgrade. 
+As a part of the upgrade to OpenEBS 4.2, the Helm chart will install all four engines regardless of the engine you used before the upgrade. 
 
 :::info
-During the upgrade, if you are only interested in Local PV Storage, you can disable Replicated PV Mayastor by setting the below option:
+During the upgrade, if you are only interested in Local PV Storage, you can disable Replicated PV Mayastor by using the below option:
 
 ```
 --set engines.replicated.mayastor.enabled=false
 ```
 :::
 
-## Update Helm Repository
-
-The OpenEBS helm chart repository is available from a different URL than before. The repository target URL needs to be updated.
-
-```
-helm repo remove openebs
-helm repo add openebs https://openebs.github.io/openebs
-helm repo update
-```
-
-## Extract Helm Values
-
-Execute the following command to capture the helm values configuration of your helm release.
-
-```
-helm get values openebs -n openebs -o yaml > old-values.yaml 
-```
-
-:::note
-If you are using a custom helm chart to use OpenEBS, compare your set of helm values against the new helm values and the values of the dependency helm charts. Refer [values.yaml](https://github.com/openebs/openebs/blob/HEAD/charts/values.yaml) for more details.
-
-If you have used helm v3.13 or above to install their chart, and not used helm's `--set` and/or `-f` options to configure their chart values, using the `-a` option with your `helm get values` command will let you capture your configuration values.
- 
-Feel free to reach out via our [communication channels](../community.md).
-:::
-
-## Local Storage Upgrade
-
-The upgrade process for Local PV Hostpath, Local PV LVM, and Local PV ZFS are largely identical, with a few changes in helm values depending on the Local PV Storage variant we are upgrading from.
-
 :::note
 Downgrades are not supported.
 :::
 
-### From 3.x to 4.2
+## Upgrade from 3.x to 4.2
```

Comment:
I think the newer steps are valid from umbrella 4.3 (mayastor 2.9) onwards, so let's update this PR accordingly.